ca/proto/ca.proto

```diff
@@ -44,6 +44,7 @@ message GenerateOCSPRequest {
   int32 reason = 3;
   int64 revokedAt = 4;
   string serial = 5;
+	// TODO(#5152): Replace issuerID with issuerNameID.
```

Comment:
Indent's a bit off.

Reply:
gofmt doesn't fix tabs vs spaces in non-go files :(